npm/ng-packs/packages/theme-shared/src/lib/components/internet-connection-status/internet-connection-status.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component } from '@angular/core';
+import { InternetConnectionService } from '../../../../../core/src/lib/services/internet-connection-service';
+
+
+
+@Component({
+  selector: 'abp-internet-status',
+  styleUrls: ['./internet-connection-status.scss'],
+  template: `
+    <div class="statusIcon" *ngIf="!internetConnection">
+      <i data-toggle="tooltip" title="The operation could not be performed. Your internet connection is not available at the moment." data-placement="top"  class="fa fa-circle text-blinking Blink"></i>

```

Comment:
title can be localizable like

`title="{{'AbpUi::InternetConnectionInfo'}}"`

We also need to add this key to Domain.Shared layer you can ask me or @mahmut-gundogdu